packages/circuit-ui/components/SidePanel/SidePanelContext.spec.tsx

```diff
@@ -43,10 +43,31 @@ import {
   type RemoveSidePanel,
   type UpdateSidePanel,
   type SidePanelContextProps,
+  type SidePanelContextItem,
 } from './SidePanelContext.js';
 
 vi.mock('../../hooks/useMedia/index.js');
 
+vi.mock('./SidePanel.js', () => ({
```

Comment:
What's the reason for mocking the SidePanel here? It would be nicer to fully test the integration, but I assume you ran into some issues?